tests/test_symbolics.py

```diff
@@ -501,6 +501,25 @@ def test_component_access():
     assert cf2 == cf1
 
 
+def test_vector_access():
```

Comment:
A test to check that some trivial operator featuring a `VectorAccess` compiles and runs would also be desirable imo

Reply:
how? it gets created and then lowered right in the middle of the CUDA/HIP compilation pipeline, so... there are tens of tests using it in the corresponding PRO PR though